docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DockerComposeExecutable.java

```diff
@@ -108,7 +108,7 @@ protected List<String> dockerComposePath() {
 
     @Value.Default
     public boolean useDockerComposeV2() {
-        return false;
+        return true;
```

Comment:
Definitely need changelog for a change like this.

Reply:
We already did this intentionally in https://github.com/palantir/docker-compose-rule/pull/729. We just forgot to change the default here.

Reply:
Eh, I still think it's useful for every human authored change to have changelog, it really helps when builds break and you can quickly see what happened.